docs/src/main/docs/about/02_introduction.adoc

```diff
@@ -0,0 +1,63 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Introduction
+:description: about Helidon
+:keywords: helidon, java, microservices, microprofile
+
+== A collection of Java libraries
+
+Helidon is a collection of Java libraries for writing microservices.
+
+There is no specific tooling or deployment model, you simply use your favorite
+ tools !
+
+NOTE: Your microservice is a plain Java application, you do not need an
+ application server.
+
+== Cloud native Java
+
+Helidon libraries interoperate with popular tools from the cloud-native space:
+
+* https://www.docker.com/[Docker]
+* https://kubernetes.io/[Kubernetes]
+* https://prometheus.io/[Prometheus]
+* https://opentracing.io/[OpenTracing]
+* https://coreos.com/etcd/[Etcd]
+
+TIP: The <<getting-started/02_base-example.adoc,Helidon Quickstart Examples>>
+ contain support for Docker and Kubernetes.
+
+== Helidon SE
+
+Helidon SE is the branded name for the functional programming style provided
+ by all Helidon libraries.
+
+NOTE: Helidon SE gives you full transparency and puts you in control. There is
+ no magic!
+
+== Helidon MP
+
+Helidon MP is the branded name for our https://microprofile.io[MicroProfile]
+ implementation.
+
+https://microprofile.io[MicroProfile] is a platform definition that is familiar
```

Comment:
There's no need to provide a link again considering there's a link in just the previous sentence. Alternatively, keep the link here and remove it from the previous sentence.